modules/cnf-du-creating-a-machine-config-manifest.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-creating-a-machine-config-manifest_{context}"]
+
+= Creating a machine config manifest
+
+To enable workload partitioning, you need to provide a `MachineConfig` manifest during installation to configure CRI-O and kubelet to know about the workload types.
```

Comment:
nit: instead of "To enable" maybe "Part of configuring workload partitioning requires you to provide..."

My point is that the high-level step information indicated that this is the current steps 1 and 2.  "To enable" might suggest that the goal is complete after applying the manifest.